### PR TITLE
CNDE-1968: Removing redundant columns

### PR DESCRIPTION
**Pull request by johneipe**
## Notes

Removing redundant and unused columns

## JIRA

- **Related story**: [CNDE-1968](https://cdc-nbs.atlassian.net/browse/CNDE-1968)

## Checklist

- [x] PR focuses on a single story.
- [ ] New unit tests added and ensured they pass.
- [ ] Service has been tested in local and it works as expected.
- [ ] Documentation has been updated for this code change (if needed).
- [ ] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [x] Bugfix
- [ ] New feature
- [ ] Breaking change

## Testing

- [ ] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [ ] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

File: db/upgrade/odse/routines/006-sp_investigation_event.sql [hunk 1]
```diff
@@ -30,7 +30,6 @@ BEGIN
         /*Complete Investigation section*/
         SELECT results.public_health_case_uid,
                results.program_jurisdiction_oid as program_jurisdiction_oid,
-               jc.code                          as jurisdiction_code,
```
Comment (upasanapattnaik-eq) [inline db/upgrade/odse/routines/006-sp_investigation_event.sql:33]: Does this have any downstream usage in post-processing?

File: db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql [hunk 2]
```diff
@@ -94,8 +94,6 @@ CREATE TABLE dbo.nrt_investigation (
     cd                                  varchar(50)                                     NULL,
     cd_desc_txt                         varchar(100)                                    NULL,
     prog_area_cd                        varchar(20)                                     NULL,
-    jurisdiction_code                   varchar(20)                                     NULL,
-    jurisdiction_code_desc_txt          varchar(255)                                    NULL,
```
Comment (sveselev) [inline db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql:98]: What about corresponded changes in the `InvestigationReporting.java`?

**Review by sveselev — CHANGES_REQUESTED** [review]

File: db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql (hunk 2 above)
Comment (johneipe) [reply db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql:98]: checking on it, when I remove it , the service is failing.

File: db/upgrade/odse/routines/006-sp_investigation_event.sql (hunk 1 above)
Comment (johneipe) [reply db/upgrade/odse/routines/006-sp_investigation_event.sql:33]: did not find any usage in the post-processing procs.

File: db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql (hunk 2 above)
Comment (sveselev) [inline db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql:98]: Why `jurisdiction_code_desc_txt` is deleted from nrt? Correspondent column is the event DTO is `jurisdictionNm` and it's not deleted from the payload.

**Review by sveselev — CHANGES_REQUESTED** [review]

File: db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql (hunk 2 above)
Comment (johneipe) [reply db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql:98]: it is working now, without deleteing the columns.

Comment (johneipe) [reply db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql:98]: jurisdiction_code_desc_txt is not used.

select count(*) from nrt_investigation;--13709
select count(*) from nrt_investigation where jurisdiction_code_desc_txt is null; ----13709

and jurisdiction_cd is the replacement for jurisdiction_code.

select sum (case when jurisdiction_cd = jurisdiction_code then 1 else 0 end) as 'j code match' from nrt_investigation ;

Comment (sveselev) [inline db/upgrade/rdb_modern/tables/007-create_nrt_investigation.sql:98]: 👍🏻

File: investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/repository/model/reporting/InvestigationReporting.java [hunk 3]
```diff
@@ -11,7 +11,6 @@
 public class InvestigationReporting {
     private Long publicHealthCaseUid;
     private Long programJurisdictionOid;
-    private String jurisdictionCode;
```
Comment (sveselev) [inline investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/repository/model/reporting/InvestigationReporting.java:14]: 👍🏻

**Review by sveselev — APPROVED** [review]
LGTM!

**Pull request merged**